src/als/ui/dialogs.py

```diff
@@ -45,17 +50,17 @@ def __init__(self, parent=None):
 
         config_to_image_save_type_mapping[config.get_image_save_format()].setChecked(True)
 
-        self._show_missing_folders()
+        self._validate_all_pathes()
 
     @log
-    def _show_missing_folders(self):
+    def _validate_all_pathes(self):
         """
         Draw a red border around text fields containing a path to a missing folder
         """
 
-        for ui_field in [self._ui.ln_work_folder_path, self._ui.ln_scan_folder_path]:
+        for ui_field in [self._ui.ln_work_folder_path, self._ui.ln_scan_folder_path, self._ui.ln_master_dark_path]:
 
-            if not Path(ui_field.text()).is_dir():
+            if not Path(ui_field.text()).exists():
```

Comment:
Sorry if I sound like a PITA, but...
I think downgrading the 2 folder tests to only existence test is a loss.

We should actually check if the 2 expected folders are folders and if the expected file is a file

Reply:
No, you're right.
I just thought that the only way, for example, to put a file in the scan folder was to edit manually the config file. But in fact changes may come from filesystem changes (i.e : the user transforms a folder into a file).
So yeah, I gonna change it.

Reply:
good spirit. We want to be as foolproof as possible

Reply:
Done